R/health_icons.R

```diff
@@ -0,0 +1,53 @@
+
+#' @rdname health_icons
+#' @export
+download_health_icons <- function(){
+  url <- "https://healthicons.org/icons.zip"
+
+  meta <- jsonlite::read_json(
+    "https://raw.githubusercontent.com/resolvetosavelives/healthicons/main/package.json"
+  )
+
+  install_icon_zip(
+    "health_icons", url, function(dl_dir) file.path(dl_dir, "icons", "svg"),
```

Comment:
I've made a small fix here, the zip contains a `__MACOSX` folder which needs careful handling.